pvlib/spa.py

```diff
@@ -468,60 +456,32 @@ def julian_ephemeris_millennium(julian_ephemeris_century):
     return jme
 
 
+@jcompile(nopython=True)
```

Comment:
Do we need to declare types in this jcompile like we do in most of the other functions in this module? I haven't kept up with numba development.

Reply:
I'm not an expert here either, but I think specifying types is not necessary in general; when using numba for JIT compiling like we do, it can infer types as needed.  

In this particular case, I did try to specify types just for consistency with the rest of the code.  However it seems that specifying the appropriate type for module-level array constants is not possible using the shorthand string notation.  It is possible by importing numba, but I didn't see a clean way to no-op the type declaration when numba isn't installed, so I opted to just leave the type signature empty here instead.  I've added a comment along these lines (should have done that originally).